freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -284,6 +289,12 @@ public int getUsernameLength() throws FreeIpaClientException {
         return config.getIpamaxusernamelength();
     }
 
+    public Config getConfig() throws FreeIpaClientException {
+        Map<String, Object> params = new HashMap<>();
```

Comment:
didn't quite follow your comment earlier about this, but it seems to work fine if I change this code to:
```
    public Config getConfig() throws FreeIpaClientException {
        Map<String, Object> params = Map.of("all", true);
        return (Config) invoke("config_show", List.of(), params, Config.class).getResult();
    }
```
which is a bit more concise

Reply:
Done